src/identityApiClient.js

```diff
@@ -80,7 +81,9 @@ export default function IdentityAPIClient(mpInstance) {
                         previousMPID,
                         callback,
                         originalIdentityApiData,
-                        method
+                        method,
+                        knownIdentities,
+                        false
```

Comment:
Is this boolean a new argument? I don't see it in the signature of `parseIdentityResponse` in Identity.js

Reply:
It's there.  
![image](https://github.com/mParticle/mparticle-web-sdk/assets/5377436/f30a605c-601a-4600-a446-fd86532472c4)